client/types/startsettings.go

```diff
@@ -44,6 +44,13 @@ type StartSettings struct {
 	// be set in this field, it will be set automatically since it is required by OpAMP protocol.
 	Capabilities protobufs.AgentCapabilities
 
+	// Defines the custom capabilities of the Agent. Each capability is a reverse FQDN with
+	// optional version information that uniquely identifies the custom capability and
+	// should match a capability specified in a supported CustomMessage. The client will
+	// automatically ignore any CustomMessage that contains a custom capability that is not
+	// specified in this field.
+	CustomCapabilities []string
```

Comment:
Do we want to specify more precisely how FQDN and optional version is encoded in the string? Perhaps link to spec for that?

Reply:
Added a link.